extensions/libarchive/CompressContent.cpp

```diff
@@ -60,10 +60,29 @@ core::Property CompressContent::EncapsulateInTar(
                           "If false, on compression the content of the FlowFile simply gets compressed, and on decompression a simple compressed content is expected.\n"
                           "true is the behaviour compatible with older MiNiFi C++ versions, false is the behaviour compatible with NiFi.")
         ->isRequired(false)->withDefaultValue<bool>(true)->build());
+core::Property CompressContent::BatchSize(
+    core::PropertyBuilder::createProperty("Batch Size")
+    ->withDescription("Maximum number of FlowFiles processed in a single session")
+    ->withDefaultValue<uint32_t>(1)->build());
 
 core::Relationship CompressContent::Success("success", "FlowFiles will be transferred to the success relationship after successfully being compressed or decompressed");
 core::Relationship CompressContent::Failure("failure", "FlowFiles will be transferred to the failure relationship if they fail to compress/decompress");
 
+std::map<std::string, CompressContent::CompressionFormat> CompressContent::compressionFormatMimeTypeMap_{
```

Comment:
These 2 maps could be constants. Also just minor comment, but maybe the naming could be changed as well to snake_case to comply with the guideline.

Reply:
made them constant, the member variable formatting is all over the place, we have both snake_case and camelCase in BinFiles, CompressContent and MergeContent (and possibly others as well), I wouldn't touch their names for now, maybe in a separate PR renaming them all at once?

Reply:
Sure that sounds okay